Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -63,7 +63,8 @@ msgstr ""
 #. translators: Keywords used in the App Store search engine to find the app.
 #. Delimit with an English comma between each keyword. Limit to 100 characters including spaces and commas.
 msgctxt "app_store_keywords"
-msgid "simplenote,note,taking,taker,notes,sync,share,simple,markdown,tag,to-do,list,lock,cloud,writing"
+msgid "notes,cloud,simple,notebook,text,to-do,todo,planner,journal,diary,sync,notepad,tag,writing,memo
+"
```

Comment:
```suggestion
msgid "notes,cloud,simple,notebook,text,to-do,todo,planner,journal,diary,sync,notepad,tag,writing,memo"
```

There's a bug in the release toolkit that no one has had time to find and fix yet and which (sometimes? always?) adds a new line at the end of a modified `msgid`. Unfortunately, this format is invalid, so we need to fix it manualy.